File: Q8.py
número1=float(input("Digite um número: "))
número2=float(input("Digite um número diferente: "))
if número1<número2:
    print(f"A ordem crescente desses números é {número1:,.6g} e {número2:,.6g}.")
else:
    print(f"A ordem crescente desses números é {número2:,.6g} e {número1:,.6g}.")

File: Q8.cs
using System;

namespace Q8
{
    class Program
    {
        static void Main(string[] args)
        {
            float número1, número2;
            String número1f, número2f;
            Console.Write("Digite um número: ");
            número1 = float.Parse(Console.ReadLine());
            Console.Write("Digite um número diferente: ");
            número2 = float.Parse(Console.ReadLine());
            número1f = string.Format("{0:#,#.######}", número1);
            número2f = string.Format("{0:#,#.######}", número2);
            if (número1 < número2)
            {
                Console.WriteLine("A ordem crescente desses números é {0} e {1}.", número1f, número2f);
            }
            else
            {
                Console.WriteLine("A ordem crescente desses números é {0} e {1}.", número2f, número1f);
            }
        }
    }
}